Makefile

```diff
@@ -15,36 +15,33 @@ REPO_PROMETHEUS_OPERATOR_LINT?=quay.io/coreos/prometheus-operator-lint
 TAG?=$(shell git rev-parse --short HEAD)
 VERSION?=$(shell cat VERSION | tr -d " \t\n\r")
 
-FIRST_GOPATH:=$(firstword $(subst :, ,$(shell go env GOPATH)))
-CONTROLLER_GEN_BINARY := $(FIRST_GOPATH)/bin/controller-gen
-GOJSONTOYAML_BINARY:=$(FIRST_GOPATH)/bin/gojsontoyaml
-JB_BINARY:=$(FIRST_GOPATH)/bin/jb
-PO_DOCGEN_BINARY:=$(FIRST_GOPATH)/bin/po-docgen
-EMBEDMD_BINARY:=$(FIRST_GOPATH)/bin/embedmd
-
 TYPES_V1_TARGET := pkg/apis/monitoring/v1/types.go
 TYPES_V1_TARGET += pkg/apis/monitoring/v1/thanos_types.go
 
+TOOLS_BIN_DIR ?= $(shell pwd)/tmp/bin
+export PATH := $(TOOLS_BIN_DIR):$(PATH)
+
+PO_DOCGEN_BINARY:=$(TOOLS_BIN_DIR)/po-docgen
+CONTROLLER_GEN_BINARY := $(TOOLS_BIN_DIR)/controller-gen
+EMBEDMD_BINARY=$(TOOLS_BIN_DIR)/embedmd
+GOBINDATA_BINARY=$(TOOLS_BIN_DIR)/go-bindata
```

Comment:
I don't think we are using `go-bindata` anymore. It was removed along with CRD management